rpm/rpm.spec

```diff
@@ -136,6 +138,7 @@ export CPPFLAGS CFLAGS LDFLAGS
     --with-vendor=meego \
     --with-external-db \
     --with-crypto=openssl \
+    --enable-zstd \
     --with-lua \
```

Comment:
Should we also keep rpm-python.spec in sync with these changes?

Reply:
Updated that spec too